src/b3d/chisight/gen3d/transition_kernels.py

```diff
@@ -371,30 +276,22 @@ def logpdf(self, new_pose, prev_pose) -> ArrayLike:
 
 @Pytree.dataclass
 class GaussianVMFPoseDriftKernel(DriftKernel):
-    """A specialized uniform drift kernel with fixed min_val and max_val, with
-    additional logics to handle the color channels jointly.
-
-    Support: [max(0.0, prev_value - max_shift), min(1.0, prev_value + max_shift)]
-    """
-
-    variance: float = Pytree.static()
+    std: float = Pytree.static()
```

Comment:
I'm pretty sure the `Pose.logpdf_gaussian_vmf_pose` takes a std, not a variance.